incubator/group/keeper_test.go

```diff
@@ -194,18 +198,163 @@ func TestCreateGroupAccount(t *testing.T) {
 	}
 }
 
+func TestCreateProposal(t *testing.T) {
+	amino := codec.New()
+	pKey, pTKey := sdk.NewKVStoreKey(params.StoreKey), sdk.NewTransientStoreKey(params.TStoreKey)
+	paramSpace := subspace.NewSubspace(amino, pKey, pTKey, group.DefaultParamspace)
+
+	groupKey := sdk.NewKVStoreKey(group.StoreKeyName)
+	k := group.NewGroupKeeper(groupKey, paramSpace, baseapp.NewRouter(), &testdata.MyAppProposal{})
+	blockTime := time.Now()
+	ctx := group.NewContext(pKey, pTKey, groupKey).WithBlockTime(blockTime)
+	defaultParams := group.DefaultParams()
+	paramSpace.SetParamSet(ctx, &defaultParams)
+
+	members := []group.Member{{
+		Address: []byte("valid-member-address"),
+		Power:   sdk.OneDec(),
+	}}
+	myGroupID, err := k.CreateGroup(ctx, []byte("valid--admin-address"), members, "test")
+	require.NoError(t, err)
+
+	policy := group.ThresholdDecisionPolicy{
+		Threshold: sdk.ZeroDec(),
+		Timout:    types.Duration{Seconds: 1},
+	}
+	accountAddr, err := k.CreateGroupAccount(ctx, []byte("valid--admin-address"), myGroupID, policy, "test")
+	require.NoError(t, err)
+
+	policy = group.ThresholdDecisionPolicy{
+		Threshold: sdk.NewDec(math.MaxInt64),
+		Timout:    types.Duration{Seconds: 1},
+	}
+	bigThresholdAddr, err := k.CreateGroupAccount(ctx, []byte("valid--admin-address"), myGroupID, policy, "test")
+	require.NoError(t, err)
```

Comment:
can you test for an error case too?

Reply:
This test is about "create proposals" only. The group account is part of the test setup only.  There are non happy path tests in `TestCreateGroupAccount` and `TestMsgCreateGroupAccountStd`